app/views/landing_page/blocks/_card.html.erb

```diff
@@ -1,16 +1,24 @@
 <%
   add_view_stylesheet("landing_page/card")
-
-  link_classes = %w(app-b-card__link govuk-link gem-print-link)
 %>
-
 <div class="app-b-card">
-  <div class="app-b-card__textbox">
-    <% block.card_content.each do |subblock| %>
+  <% block.card_content.each do |subblock| %>
+    <% if subblock.data["type"] == "heading" %>
```

Comment:
I like this. I think this logic will make it easier to move the image logic here as well if we wanted, so it too could be defined as a block.